_sass/_components/layout.scss

```diff
@@ -48,11 +48,51 @@
     color: $color-white;
   }
 
-  a {
+  a
+   {
     color: $color-bright;
 
     &:hover {
       color: $color-bright-hover;
     }
   }
+  .usa-button,
+  .usa-button-primary,
+  .usa-button:visited,
+  .usa-button-primary:visited,
+  button,
```

Comment:
No, this is a separate thing from that issue - this cascades light link and button styles when you're using a dark background.